orttraining/orttraining/python/postprocess.py

```diff
@@ -0,0 +1,332 @@
+import sys
```

Comment:
Lara's PR has 259 lines for postprocess.py. This has  332 lines. What is the difference?

Reply:
rebase error that includes additional lines that are removed later. fixed